main_test.go

```diff
@@ -233,3 +235,20 @@ func setEnvVar(t *testing.T, key, value string) (cleanUp func()) {
 		assert.NoError(t, os.Unsetenv(key))
 	}
 }
+
+// Validate Cli subcommands aliases for duplication.
```

Comment:
Validate Cli subcommands aliases for duplication.
-->
Validate that all CLI commands' aliases are unique, and that two commands don't use the same alias.